wandb/apis/public/runs.py

```diff
@@ -505,26 +585,34 @@ def load(self, force=False):
                 if self._attrs.get("summaryMetrics")
                 else {}
             )
-        except json.decoder.JSONDecodeError:
-            # ignore invalid utf-8 or control characters
-            self._attrs["summaryMetrics"] = json.loads(
-                self._attrs["summaryMetrics"],
-                strict=False,
+        except (json.decoder.JSONDecodeError, KeyError):
```

Comment:
Doesn't the condition `if self._attrs.get("summaryMetrics")` guarantee that `self._attrs["summaryMetrics"]` doesn't raise a `KeyError`? I'm not sure these updates are needed.